reactioncommerce/reaction
# Fixed browser console displaying errors when email/accounts action panel is viewed

Resolves #3219,

I noticed that `noDataText` prop for React-Table was passed an arrow function that returns a HTML span element as seen below,
```
noDataText={() => <span className="sortableTable-noDataText">{this.props.noDataMessage}</span>}
```
So I resolved the bug by declaring a function `displayNoResultsFound` and put the definition of the arrow function passed into it there rather and invoked it as seen below, 
```
  const displayNoResultsFound = () => {
        const displayText = <span className="sortableTable-noDataText">{this.props.noDataMessage}</span>;
        return displayText;
    };
.....
.....
.....
noDataText={displayNoResultsFound()}
```

#### HOW TO TEST

1. Sign in as admin
1. Refresh page
1. Click on Accounts setting in the side action panel
1. Observe no error in browser console`
